code/modules/uplink/uplink_items.dm

```diff
@@ -2099,3 +2099,10 @@ datum/uplink_item/role_restricted/superior_honkrender
 	item = /obj/item/storage/fancy/cigarettes/cigpack_syndicate
 	cost = 2
 	illegal_tech = FALSE
+
+/datum/uplink_item/role_restricted/traitorous_cloner
+	name = "Defective Cloner"
+	desc = "A box that cointains boards and tools to setup defective clone machinery for making defective clones."
+	item = /obj/item/storage/box/syndicate/traitorous_cloner
+	cost = 4
+	restricted_roles = list("Geneticist", "Chief Medical Officer")
```

Comment:
Why not MD too? They use cloning a lot.

Reply:
Because these are only main jobs that are supposed to sit and fuck beside the cloner.